app/containers/SearchBox/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { TextInputProps, View } from 'react-native';
+
+import { useTheme } from '../../theme';
+import I18n from '../../i18n';
+import FormTextInput from '../TextInput/FormTextInput';
+
+export interface ISearchBox extends TextInputProps {
+	onCancelSearch: () => void;
+}
+
+const SearchBox = ({ onCancelSearch, onChangeText, onSubmitEditing, value, testID }: ISearchBox): JSX.Element => {
+	const { colors, theme } = useTheme();
+	const background = theme === 'light' ? colors.backgroundColor : colors.searchboxBackground;
+	const inputStyle = {
+		borderWidth: 2,
+		paddingVertical: 0,
+		borderColor: colors.searchboxBackground,
+		backgroundColor: background,
+		color: colors.auxiliaryTintColor
+	};
```

Comment:
```suggestion
	const inputStyle = {
		// borderWidth: 2,
		paddingVertical: 0,
		// borderColor: colors.searchboxBackground,
		// backgroundColor: background,
		// color: colors.auxiliaryTintColor
	};
```

The ones I commented you can just remove and inherit from TextInput.
I removed `paddingVertical` and I couldn't find any difference.